spec/const3.dd

```diff
@@ -397,27 +397,98 @@ $(H2 $(LNAME2 const_member_functions, Const Member Functions))
 	this reference.
 	)
 
+$(H2 $(LNAME2 combining_qualifiers, Combining Qualifiers))
+
+$(P More than one qualifier may apply to a type. The order of application is
+irrelevant, for example given an unqualified type `T`, $(D const shared T) and
+$(D shared const T) are the same type. For that reason, this document depicts
+qualifier combinations without parentheses unless necessary and in alphabetic
+order.)
+
+$(P Applying a qualifier to a type that already has that qualifier is legal but
+has no effect, e.g. given an unqualified type `T`, $(D shared(const shared T))
+yields the type $(D const shared T).)
+
+$(P Applying the `immutable` qualifier to any type (qualified or not) results in
+$(D immutable T). Applying any qualifier to $(D immutable T) results in $(D
+immutable T). This makes `immutable` a fixed point of qualifier combinations and
+makes types such as $(D const(immutable(shared T))) impossible to create.)
+
+$(P Assuming `T` is an unqualified type, the graph below illustrates how
+qualifiers combine (combinations with `immutable` are omitted). For each node,
+applying the qualifier labeling the edge leads to the resulting type.)
+
+$(COMMENT To generate qualifier-combinations.svg, run "dot -Tsvg t.dot
+>qualifier-conversions.svg" against a file t.dot containing the following:
+
+digraph "Qualifer Combinations" {
+    rankdir="TB";
+    "T" -> "const T" [label="const"];
+    "T" -> "inout T" [label="inout"];
+    "T" -> "shared T" [label="shared"];
+    "const T" -> "const inout T" [label="inout"];
+    "const T" -> "const shared T" [label="shared"];
+    "inout T" -> "const inout T" [label="const"];
+    "inout T" -> "inout shared T" [label="shared"];
+    "shared T" -> "const shared T" [label="const"];
+    "shared T" -> "inout shared T" [label="inout"];
+    "const inout T" -> "const inout shared T" [label="shared"];
+    "const shared T" -> "const inout shared T" [label="inout"];
+    "inout shared T" -> "const inout shared T" [label="const"];
+}
+)
 
-$(H2 $(LNAME2 implicit_conversions, Implicit Conversions))
+<center>
```

Comment:
FWIW
1 ) [`center` isn't supported in HTML5](https://www.w3schools.com/tags/tag_center.asp), there's `$(DIVC center)` but please see point (2)
2) The spec is also generated to PDF/ebook - there are already many issues with people using HTML (see [Bugzilla](https://issues.dlang.org/buglist.cgi?component=dlang.org&list_id=215655&product=D&resolution=---)). 

This is how it will look in the PDF:

![image](https://user-images.githubusercontent.com/4370550/27774329-a0ff00ae-5f90-11e7-98da-37b855d463ac.png)

Unfortunately there's no precedent of images in the spec and you most likely need to add the Ddoc macros for images yourself.